backend/src/v2/compiler/argocompiler/dag.go

```diff
@@ -631,7 +631,9 @@ func addImplicitDependencies(dagSpec *pipelinespec.DagSpec) error {
 					return wrap(err)
 				}
 			case *pipelinespec.TaskInputsSpec_InputParameterSpec_TaskFinalStatus_:
-				return wrap(fmt.Errorf("task final status not supported yet"))
+				if err := addDep(input.GetTaskFinalStatus().GetProducerTask()); err != nil {
```

Comment:
Is `addDep` required since my understanding is that a task that uses `dsl.PipelineTaskFinalStatus` must be an exit task? Exit tasks are handled specially with Argo Workflow exit handlers.